sdk/playwrighttesting/microsoft-playwright-testing/src/utils/reporterUtils.ts

```diff
@@ -317,19 +318,32 @@ class ReporterUtils {
     },
   };
 
-  public static isTimeGreaterThanCurrentPlus10Minutes(isoString: string): boolean {
-    // Parse ISO 8601 string into a timestamp
-    const timestampFromIsoString: number = Date.parse(isoString);
-    // Calculate the current timestamp plus 10 minutes
-    const currentTimestampPlus10Minutes: number = Date.now() + 10 * 60 * 1000;
-    // Compare the timestamps
-    return timestampFromIsoString > currentTimestampPlus10Minutes;
+  public static isTimeGreaterThanCurrentPlus10Minutes(sasUri: string): boolean {
+    try {
+      const url = new URL(sasUri);
+      const params = new URLSearchParams(url.search);
+      const expiryTime = params.get("se"); // 'se' is the query parameter for the expiry time
+      reporterLogger.error(`\nExpiryTimeFromSasUri: ${expiryTime}`);
```

Comment:
debug

Reply:
This will output too many logs, we should have a debug log only when the sas is close to expiry